geocropper/geocropper.py

```diff
@@ -975,5 +975,38 @@ def move_crops_containing_locations(csv_path, source_dir, target_dir, outside_cr
                         utils.move_crops_containing_locations(csv_path, group, (target_dir / group.name))
 
 
-def stack_trimmed_images():
-    pass
+def stack_trimmed_images(root_dir, speckle, output_dir=None):

```

Comment:
Usually I named it source_dir instead of root_dir. We should change it here to source_dir as well.